posthog/temporal/tests/data_modeling/test_run_workflow.py

```diff
@@ -1147,3 +1148,87 @@ async def test_create_job_model_activity_cleans_up_running_jobs(activity_environ
     assert new_job.status == DataModelingJob.Status.RUNNING
     assert new_job.workflow_id == "new-workflow"
     assert new_job.workflow_run_id == "new-run"
+
+
+@pytest.mark.asyncio
```

Comment:
nit: this isn't needed as we have `pytestmark = [pytest.mark.asyncio, pytest.mark.django_db]` higher up in the file